bentoml/yatai/deployment/aws_lambda/lambda_app.py

```diff
@@ -66,7 +66,13 @@ def api_func(event, context):  # pylint: disable=unused-argument
     """
 
     if type(event) is dict and "headers" in event and "body" in event:
-        return bento_service_api.handle_aws_lambda_event(event)
+        print(f'Got prediction request with body "{event["body"]}"')
+        prediction = bento_service_api.handle_aws_lambda_event(event)
+        print(
```

Comment:
This is great.  

We can take one step further and make this into JSON format with the input and output.  Users can stream the logs out to ElasticSearch for searching and/or use the result for training later on. 

You can check out the current prediction log format in the BentoAPIServer for reference: https://github.com/bentoml/BentoML/blob/a134e3a27b2158a650190bcffc9e7540a1f49b59/bentoml/server/bento_api_server.py#L293